test/tools/auth0/handlers/logStreams.test.ts

```diff
@@ -60,11 +61,46 @@ describe('#logStreams handler', () => {
       const handler = new logStreamsHandler({ client: auth0ApiClientMock });
       const data = await handler.load();
 
-      const expectedLogStreams = logStreams.filter((logStream) => {
-        return logStream.status !== 'suspended';
+      expect(data).to.deep.equal({
+        logStreams: [
+          {
+            id: 'log-stream-1',
+            name: 'Splunky',
+            type: 'splunk',
+            status: 'paused',
+            sink: {
+              splunkDomain: 'test-splunk.com',
+              splunkPort: '8089',
+              splunkToken: '7b838bd0-028e-4d78-a82c-3564a2007770',
+              splunkSecure: false,
+            },
+          },
+          {
+            id: 'log-stream-2',
+            name: 'HTTP Log Stream',
+            type: 'http',
+            status: 'active',
+            sink: {
+              httpAuthorization: '_VALUE_NOT_SHOWN_', // secret obfuscated
+              httpContentFormat: 'JSONLINES',
+              httpContentType: 'application/json',
+              httpEndpoint: 'https://example.com/test',
+            },
+          },
+          {
+            id: 'log-stream-3',
+            name: 'Suspended HTTP Log Stream',
+            type: 'http',
+            // status property omitted if suspended
+            sink: {
+              httpAuthorization: '_VALUE_NOT_SHOWN_', // secret obfuscated
+              httpContentFormat: 'JSONLINES',
+              httpContentType: 'application/json',
+              httpEndpoint: 'https://suspended.com/logs',
+            },
+          },
+        ],
```

Comment:
The difference between the output of the function and the `mockLogStreams` input is enough where I'd prefer to not write cryptic code to generate the expected value. Although more lines of code, it's clearer what the output is IMO.